src/Provider/RepositoryRefundedUnitTotalProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Provider;
+
+use Sylius\Component\Core\Model\OrderItemUnitInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Webmozart\Assert\Assert;
+
+final class RepositoryRefundedUnitTotalProvider implements RefundedUnitTotalProviderInterface
```

Comment:
Honestly, by this class name, I am unable to tell what it exactly does. I guess that the `repository` word is unnecessary.